src/OrganisationUnitTree/useContent.js

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react'
+import { useChildReloadIds } from './useChildReloadIds'
+
+export const useContent = ({
+    open,
+    path,
+    props,
+    error,
+    loading,
+    children,
+    hasChildren,
+    OrganisationUnitNode,
+    childrenIds,
+    orgUnitsPathsToInclude,
+    idsThatShouldBeReloaded,
+}) => {
+    const childIds = useChildReloadIds(childrenIds, idsThatShouldBeReloaded)
+
+    return useMemo(() => {
+        const showChildren = !loading && !error && open
+
+        if (!(hasChildren || children.length)) {
+            return null
+        }
+
+        if (!showChildren) {
+            // @TODO Ask Joe Cooper
+            return (
+                <span
+                    style={{
+                        padding: '4px 4px 10px 30px',
+                        display: 'block',
+                    }}
+                >
+                    Loading...
+                </span>
+            )
```

Comment:
@Joe we need to think about how to style this:
https://deploy-preview-10--dhis2-ui-widgets.netlify.com/?path=/story/organisationunittree--loading